app/src/main/res/layout/dialog_progress.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout
```

Comment:
Please, use **flat** hierarchy: all views should be child to a parent view (we can use `ConstraintLayout` or `RelativeLayout` as a parent)